### ISSUE TITLE
More helpful errors

### ISSUE THREAD

**Issue by Enivex** [mitex-rs/mitex#116]
I have a large, complicated document that I didn't expect would compile, but I was hoping MiTeX would tell me _what_ it couldn't handle. Instead I was met with this error

![image](https://github.com/mitex-rs/mitex/assets/2191754/2c03af13-e07d-4633-867e-343f8b89c82d)

It would be great if it would give some details :)

**Comment by dflock** [comment]
Yeah, a verbose flag, or even just a stacktrace - anything really, would be much more helpful.
